homeassistant/components/notify/__init__.py

```diff
@@ -18,6 +18,7 @@
 from homeassistant.util import slugify
 
 DOMAIN = "notify"
+GROUP_DOMAIN = "notifygroup"
```

Comment:
This breaks the rule that you only have config in `DOMAIN:` or `DOMAIN *:`. We cannot work around that in the loader because we can't know which components exist without trying to preload everything.